helpers/test/basic.test.js

```diff
@@ -122,6 +122,7 @@ test('formatHttpRequest and formatHttpResponse should return a valid ecs object'
         })
       }
     )
+    req.setHeader('connection', 'close')
```

Comment:
Perhaps instead add `connection: 'close'` to the `headers: { ... }` object above on line 110?

Reply:
https://github.com/elastic/ecs-logging-nodejs/pull/132/commits/44da905e31f900162b8471916c4006b7edd46a53